cosi.proto

```diff
@@ -152,19 +158,15 @@ message ProvisionerGrantBucketAccessRequest {
 }
 
 message ProvisionerGrantBucketAccessResponse {
-    // This field is OPTIONAL
     // This is the account that is being provided access. This will
-    // be required later to revoke access. 
+    // be required later to revoke access.
+    // This field is REQUIRED.
     string principal = 1;
-    
-    // This field is OPTIONAL
-    // Contents of the credential file, ex: aws access key id and secret, etc.
-    string credentials_file_contents = 2;
-    
-    // This field is OPTIONAL
-    // Path where the credential file will be mounted.
-    string credentials_file_path = 3;
-} 
+
+    // This is all the file information for access credentials
+    // in the format <filename,contents>
+    map<string,string> credentials = 2;
```

Comment:
Please indicate if this is optional. Also document any general expectations of the k-v

Reply:
I attempted to that with the comment:
in the format <filename,contents>

key = filename
value = contents

Will need to add clarification that it needs to be file path not just file name.  What other clarification are you looking for?

Reply:
Please mark this optional.

Reply:
Done.